views/pages/fixture_editor.js

```diff
@@ -991,37 +965,68 @@ function numberInput(modelKey, options = {}) {
 }
 
 /**
- * @param {!object} property The JSON schema property info.
- * @returns {?number} The lowest possible value the property can be set to, or null if there's no minimum.
+ * @param {!string} modelKey The Vue reference used in v-model.
+ * @param {!InputOptions} options Additional optional parameters.
+ * @returns {!string} The HTML attribute(s) declaring a minimum and/or maximum value, or an empty string.
  */
-function getMininium(property) {
-  if (property) {
-    if (`minimum` in property) {
-      return property.minimum;
-    }
-    if (`exclusiveMinimum` in property) {
-      return property.exclusiveMinimum;
+function getMinMaxAttributes(modelKey, options) {
+  let html = ``;
+
+  for (const minMax of [`in`, `ax`]) {
+    let minMaxValue = null;
+
+    if (options.property) {
+      if (`m${minMax}imum` in options.property) {
+        minMaxValue = options.property[`m${minMax}imum`];
+      }
+      if (`exclusiveM${minMax}imum` in options.property) {
+        minMaxValue = options.property[`exclusiveM${minMax}imum`];
+
+        // integer values have a discrete minimum, floats not
+        if (options.property.type === `integer`) {
+          minMaxValue++;
+        }
+        else {
+          // this could be validated in future
+          html += ` data-exclusive-m${minMax}imum="${minMaxValue}"`;
+        }
+      }
     }
+
+    html += getMinMaxAttribute(modelKey, `m${minMax}`, minMaxValue);
   }
 
-  return null;
+  return html;
 }
 
 /**
- * @param {!object} property The JSON schema property info.
- * @returns {?number} The highest possible value the property can be set to, or null if there's no maximum.
+ * @param {!string} modelKey The Vue reference used in v-model.
+ * @param {'min'|'max'} minMax Whether a minimum or maximum is desired.
+ * @param {!number} minMaxValue The numeric value of the minimum or maximum.
+ * @returns {!string} The HTML attribute declaring a minimum or maximum value, or an empty string.
  */
-function getMaximum(property) {
-  if (property) {
-    if (`maximum` in property) {
-      return property.maximum;
+function getMinMaxAttribute(modelKey, minMax, minMaxValue) {
+  let html = ``;
+
+  // when we have a minimum value, we have to identify a maximum counterpart of a range
+  const pattern = minMax === `min` ? /([mM])ax/i : /([mM])in/i;
+  const replaceString = minMax === `min` ? `$1in` : `$1ax`;
+
+  if (minMaxValue !== null) {
```

Comment:
This `null` check can then also be removed.

*Note:* The JSDoc comment forbids `null` values for `minMaxValue` anyway.

Reply:
`minMaxValue` is `null` when the property defines no minimum or maximum. Anyway, I'll move the condition to the `getMinMaxAttributes` function.